frontend/src/test/scala/io/udash/ApplicationTest.scala

```diff
@@ -11,7 +13,7 @@ class ApplicationTest extends UdashFrontendTest {
     val vpRegistry: TestViewPresenterRegistry = new TestViewPresenterRegistry(Map.empty, viewPresenter)
     val app = new Application[TestState](routing, vpRegistry, RootState, urlProvider)
 
-    app.run(null)
+    app.run(div().render)
```

Comment:
All replacements of "div().render" have meaning of "empty component", so it can be wrapped in explicit method.